README.md

```diff
@@ -1,85 +1,88 @@
+
+## Installation
+
+To install Helmbin, follow these steps:
+
+1. Download the latest release from the [Helmbin releases page](https://github.com/replicatedhq/helmbin/releases).
+2. Extract the downloaded archive to a directory of your choice.
+3. Add the directory containing the Helmbin binary to your system's `PATH` variable.
 
 ## Building
 
+If you want to build the binary on your own you can, after cloning this repository, run:
+
 ```bash
 $ make build
-go build -gcflags "all=-trimpath=/home/ethan/go/src/github.com/replicatedhq" -asmflags "all=-trimpath=/home/ethan/go/src/github.com/replicatedhq" -ldflags " -X main.goos=linux -X main.goarch=amd64 -X main.gitCommit=1a6e487bb4bcb5049c448983758912afbdb9d1c2 -X main.buildDate=2023-05-24T21:42:34Z " -tags='' -o bin/helmbin ./cmd/helmbin
 ```
 
+The compiled binary will be then placed under the `bin/` subdirectory.
+
 ## Running
 
+By default `Helmbin` deploys [Kots](https://kots.io/) Helm Chart, support for additional Helm Charts are coming. Once you have Helmbin installed you can run it in either foreground or as a `systemd` unit.
+
+### Running on foreground
+
+To run it in foreground you can use:
+
 ```bash
-./bin/helmbin server  --help
-Runs a controller+worker node
-
-Usage:
-  helmbin run [flags]
-  helmbin run [command]
-
-Available Commands:
-  controller  Runs a controller node
-  worker      Runs a worker node
-
-Flags:
-  -c, --config string       k0s config file, use '-' to read the config from stdin (default "/etc/k0s/k0s.yaml")
-      --data-dir string     Data Directory. DO NOT CHANGE for an existing setup, things will break! (default "/var/lib/replicated")
-  -d, --debug               Debug logging (default: false)
-      --enable-worker       enable worker (default true)
-  -h, --help                help for run
-      --no-taints           disable default taints for controller node (default true)
-      --token-file string   Path to the file containing join-token.
-
-Use "helmbin run [command] --help" for more information about a command.
+$ sudo helmbin run
 ```
 
+Be aware that all logs are going to be printed in the current terminal and Helmbin tends to be very verbose. After a few seconds you should be able to access [Kots](https://kots.io/) by accessing the Virtual Machine IP Address on port `8800` (e.g. http://192.168.0.1:8800/)
+
+### Running as a systemd unit
+
+Systemd is a system initialization and service management framework for Linux that provides a more efficient and centralized approach to managing and controlling system processes. To install and run Helmbin as a systemd unit you can run:
+
+```bash
+$ sudo helmbin install
+```
+
+This will create a system unit service called `k0scontroller`. Helmbin can then be stopped and started on demand with the following commands:
+
+```bash
+$ sudo helmbin stop
+$ sudo helmbin start
+```
+
+You can enable Helmbin startup during boot by running:
+
+```bash
+$ sudo systemctl enable k0scontroller.service
```

Comment:
This should be "branded" with the name of the helm chart that's packaged.
So, if we are defaulting to `kotsadm`, should should be `sudo systemctl enable kotsadm.service`

Reply:
1. can we build this into the install command?
2. in the future i think we should write our own install command so we can brand things rather than passthrough to the k0s command.